pallets/summary/Cargo.toml

```diff
@@ -40,7 +40,7 @@ assert_matches = "1.3.0"
 
 [features]
 migrate = []
-default = ["std", "pallet-session/historical"]
```

Comment:
This is a change on a different pallet.
Is it ok to remove this from the feature list of the pallet?